features/admin.userstores.v1/pages/user-stores.tsx

```diff
@@ -231,15 +281,42 @@ const UserStores: FunctionComponent<UserStoresPageInterface> = (
                     <Show
                         when={ featureConfig?.userStores?.scopes?.create }
                     >
-                        <PrimaryButton
-                            onClick={ () => {
-                                history.push(AppConstants.getPaths().get("USERSTORE_TEMPLATES"));
-                            } }
-                            data-testid={ `${ testId }-list-layout-add-button` }
-                        >
-                            <Icon name="add"/>
-                            { t("userstores:pageLayout.list.primaryAction") }
-                        </PrimaryButton>
+                        { disabledFeatures?.includes("userStores.type.remote")
+                            ? (
+                                <PrimaryButton
+                                    onClick={ () => {
+                                        history.push(AppConstants.getPaths().get("USERSTORE_TEMPLATES"));
+                                    } }
+                                    data-testid={ `${ testId }-list-layout-add-button` }
+                                >
+                                    <Icon name="add"/>
+                                    { t("userstores:pageLayout.list.primaryAction") }
+                                </PrimaryButton>
+                            ) : (
+                                <Dropdown
+                                    data-componentid={ `${ testId }-add-user-dropdown` }
+                                    direction="left"
+                                    floating
+                                    icon={ null }
+                                    trigger={ addUserDropdownTrigger }
+                                >
+                                    <Dropdown.Menu>
+                                        { getAddUserOptions().map((option: {
+                                            "data-componentid": string;
+                                            key: number;
+                                            text: string;
+                                            value: UserAccountTypes;
+                                        }) => (
```

Comment:
Can't we use `DropdownItemProps` as the type for `option` here?
And check whether using `UserAccountTypes` type is expected here.

Reply:
Added with 0bc853333b14313614be9c0e79c2dd4a88d30ea0.